lib/language_pack/helpers/rails_runner.rb

```diff
@@ -98,16 +99,34 @@ def command
   end
 
   private
-
     def call
       topic("Detecting rails configuration")
-      out = run(command, user_env: true)
-      @success = $?.success?
+      out = execute_command!
+
       if @debug
         puts "$ #{command}"
         puts out
       end
       out
     end
+
+    def execute_command!
+      out = String.new
+      Timeout.timeout(@timeout) do
+        pipe(command, user_env: true, silent: true, buffer: out)
+      end
+      @success = $?.success?
+
+      unless @success
+        warn("Detecting rails configuration failed\nset HEROKU_DEBUG_RAILS_RUNNER=1 to debug")
```

Comment:
It looks like this will output even if you're debugging?

Reply:
Good call. I changed it so i'm only modifying the message if debug is not being used.